src/test/http-mocks/claim-store.ts

```diff
@@ -401,10 +401,10 @@ export function resolveRetrieveClaimByExternalIdTo404HttpCode (reason: string =
     .reply(HttpStatus.NOT_FOUND, reason)
 }
 
-export function resolveRetrieveByClaimantId (claimOverride?: object) {
+export function resolveRetrieveByClaimantId (claim: object = sampleClaimObj, claimOverride?: object) {
```

Comment:
are you doing this to be able to use it for other claim objects if needed in future?

Reply:
I did this so that I didn't have to modify all of the (hundreds?) of route tests currently using that method with the old signature

Reply:
But yes, it thought it does not make so much sense to keep duplicating methods when the only difference is a single object, it's better to just pass it in